extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -176,6 +177,24 @@ define(
 		$('.editpage-widemode-trigger').click(editorModeChange);
 	}
 
+	function launchEditorPreview(){
+		var editor_value = ace.getContent();
+		var wiki_url = mw.config.get('wgServer').split('://')[1];
+		var template_name = mw.config.get('wgPageName').split(':')[1];
+
+		var encode = function(str) {
+			return encodeURI(str).replace(/'/g, '%27');
+		}
+
+		$('<form action="http://localhost:8080/api/entrypoint/" method="POST">')
```

Comment:
If you use jQuery .val to set values of those hidden fields then you wouldn't have to deal with encoding altogether. Something like:
```
$inputEditorValue = $('input').val(editorValue).appendTo($form);
```